docs/source/cn/_redirects.yml

```diff
@@ -0,0 +1,11 @@
+# Move "how-to" pages to the guides/ folder
+how-to-cache: guides/manage-cache
+how-to-discussions-and-pull-requests: guides/community
+how-to-downstream: guides/download
+how-to-inference: guides/inference
+how-to-manage: guides/repository
+how-to-model-cards: guides/model-cards
+how-to-upstream: guides/upload
+search-the-hub: guides/search
+guides/manage_spaces: guides/manage-spaces
+package_reference/inference_api: package_reference/inference_client
```

Comment:
This file is used in the English version of the docs to make legacy urls still compatible with the current documentation (when we renamed some files). So for simplified Chinese it is not necessary to have it given it is a brand new documentation.

Reply:
File deleted @Wauplin